internal/entities/user_management/user/core.go

```diff
@@ -1 +1,81 @@
 package user
+
+import (
+	"errors"
+	"github.com/freshpay/internal/config"
+	"github.com/freshpay/internal/entities/user_management/session"
+	"github.com/freshpay/internal/entities/user_management/utilities"
+	"github.com/freshpay/internal/entities/user_management/wallet"
+
+	//"github.com/freshpay/internal/entities/user_management/wallet"
+)
+
+
+
+func VerifyPhoneNumber(phoneNumber string) bool{
+	if phoneNumber=="1"{
+		return false
+	}
+	return true
+}
+//SignUp will be used to create a user on signup
+func SignUp(user *Detail) (err error){
+	phoneNumber :=user.PhoneNumber
+
+	/*
+	    Make sure PhoneNumber doesn't exist
+	 */
+	var userTemp Detail
+	err=GetUserByPhoneNumber(&userTemp,phoneNumber)
+	if err==nil{
+		err=errors.New("Phone Number is already registered")
+		return err
+	}
+	if !VerifyPhoneNumber(phoneNumber){
```

Comment:
why are verifying otp in signup flow? 
There will be separate api for otp verification.